src/rt-app.c

```diff
@@ -430,13 +430,15 @@ int run(int ind,
 	int nbevents = pdata->nbevents;
 	int i, lock = 0;
 	unsigned long perf = 0;
+	char type_name[64];
 
 	for (i = 0; i < nbevents; i++)
 	{
 		if (!continue_running && !lock)
 			return perf;
 
-		log_debug("[%d] runs events %d type %d ", ind, i, events[i].type);
+		log_debug("[%d] runs events %d type %d %s ", ind, i, events[i].type,
+			  ( resource_to_string(events[i].type, type_name), type_name) );
```

Comment:
Is the event type number still needed now that you have added the name ?

Reply:
IIRC, the problems I was debugging in playing back complex rt-app scripts were made easier by knowing what the event index was as well as the type, so I would quite like to keep if possible.

Reply:
I'm going to check this is actually ever meaningful and take it out if not.

Reply:
This is indeed bogus